build/tasks/format.js

```diff
@@ -25,6 +27,10 @@ function format(file) {
     return true;
   }
 
+  if (process.env.CI !== "true") {
+    notify.skip(chalk.gray(file));
+  }
+
```

Comment:
This can be removed as `precommit.js` takes care of notifying about skipped files.

Reply:
got it!